pkg/scheduler/internal/queue/scheduling_queue_test.go

```diff
@@ -1506,6 +1506,124 @@ func TestPriorityQueue_MoveAllToActiveOrBackoffQueue(t *testing.T) {
 	}
 }
 
+// This unit test can only exist when SchedulerQueueingHints feature gate was dsiabled by default.
+// TODO: it will be removed after SchedulingQueueHint feature gate is enabled by default.
+func TestPriorityQueue_MoveAllToActiveOrBackoffQueueWithOutQueueingHint(t *testing.T) {
```

Comment:
Let's disable the feature gate explicitly.

```suggestion
func TestPriorityQueue_MoveAllToActiveOrBackoffQueueWithOutQueueingHint(t *testing.T) {
defer featuregatetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, features.SchedulerQueueingHints, false)()
```

Reply:
Done